analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ParameterNameMatchesOriginal.cs

```diff
@@ -93,4 +91,53 @@ public partial interface IParameterNamesInPartialMethods
         {
         }
     }
+
+    public interface IGenericInterface<A>
```

Comment:
We can also add a test case with a method without parameters, as there is no such example.